### DISCUSSION TITLE
Searching doesn't include the description field?

### DISCUSSION THREAD

**Discussion by jbwhit** [massCodeIO/masscode.io#27] [Q&A] [answered]
I was looking to see if I should port my collection of txt files to massCode, and in testing it out, I found that the descriptions weren't searched. I'll give you an example, I wrote

title: "Pandas timestamp string"
description: "This is how to format a timestamp string" 
and the code snippet body:

```python
pd.Timestamp('now').strftime("%Y%m%d%H%M%S%f")
```

I would have expected if I were to search "format" it would have found that snippet, but no. Is this the intended behavior?

**Comment by antonreshetov** [comment]
The search is done through a third-party service https://www.algolia.com. Indexing takes place once every 24 hours.

**Reply by antonreshetov** [reply]
Also, description from frontmatter is meta data for SEO, it is not a descriptive part of the snippet category page. The descriptive part should be added below the snippet title. Just follow the examples of snippets already created.

**Comment by jbwhit** [comment]
I can move the description of the snippet from the description to the body of the Snippet. Having done that I can verify that works for my above use case. So I think we can close the question. Thank you for the prompt reply!

However, I have to admit almost everything else in your answer confuses me. When I run massCode on my laptop, why would algolia be used to search my snippets locally? But it's only indexed once per 24 hours? I think both of these things probably aren't true as it can find edits that I've made to my snippets immediately.

Also, why would SEO be involved with my personal snippets? I'm not looking for any search engine access to what I've written anyway. Finally, when you say description from front matter is not a descriptive part of the snippet category page. I am not writing or editing anything like a category page, I'm talking about a snippet itself.

**Reply by antonreshetov** [reply]
Look carefully at where you started the issue/discussion. This is a repo for masscode.io

**Comment by jbwhit** [comment, marked as answer]
I was talking about the app massCode, not massCode.io the website -- so my question did not make sense on multiple levels in this context.

**Reply by antonreshetov** [reply]
You are in the wrong repository, is that clear?

**Reply by jbwhit** [reply]
Crystal clear. I closed my question with this answer which stated that my original question does not make sense because it's talking about the app and not the website. Since this is the website repo, my question does not make sense in this context. So I closed the question.
